apps/tangle-website/src/libs/notion/notion.ts

```diff
@@ -0,0 +1,209 @@
+import { Client } from '@notionhq/client';
```

Comment:
Please remove all mentions of blog related functionality. It will not be needed here.